clarify use of file:// schema (#6477)
EnterpriseDB/docs: advocacy_docs/edb-postgres-ai/ai-accelerator/pgfs/functions.mdx

> The create_strorage_location function takes a name for the storage location and then a URL for the location. The URL should be prefixed with `s3:` for an S3-compatible bucket or `file:` for a local file system.

Reviewer: ```suggestion
The create_strorage_location function takes a name for the storage location and then a URL for the location. The URL should be prefixed with `s3:` for a S3-compatible bucket or `file:` for a local file system.
```

```sql
select pgfs.create_storage_location('my_file_storage', 'file:///tmp/my_path',  NULL,  '{}'::json,    '{}'::json );
```

When using the `file:` schema, an absolute path has to be provided (starting with `/`, e.g. `/tmp/my_path`). So together with the schema indicator `file://` we have three slashes at the beginning of the path.

> The function also takes an optional `msl_id` parameter, which is not used. The function also takes an  `options` parameter and credentials parameter. They are not optional and should be passed as empty JSON objects if not used.

Reviewer: ```suggestion
The function also takes an optional `msl_id` parameter, which is not used. It also requires `options` and `credentials` parameters; if unused, they must be passed as empty JSON objects.
```
